app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>User displayed here </h1>
\ No newline at end of file
+<h1>Users Index action</h1>
+
+<% @users.each do |person| %>
+<%= link_to "users/#{person.id}" do%>
+<%= render partial: "shared/user_card", :locals => {person: person} %>
```

Comment:
[ OPTIONAL] 
- Great job makin use of partials. A better approach while using partials is not to use @users.each. You can consider the code snippet below:

 _**_app/views/users/_user.html.erb_**_

   
   `  HTML Code Here displaying`
        `user.photo, user.name and user.posts_counter`


     **_app/views/users/index.html.erb_**

    `<%= render @users %>`